### PR TITLE
OpenShift Policy integration

### PR DESCRIPTION
**Pull request by pweil-**
Not done: PSC level SELinux

Noticed these tests are failing on the branch: https://gist.github.com/pweil-/bf44f0664349c3b5c5b3

### CODE REVIEW

**Comment by pmorie** [comment]
Looking good so far :)

File: Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation.go [hunk 1]
```diff
@@ -2060,7 +2060,44 @@ func ValidateSecurityContextConstraints(scc *api.SecurityContextConstraints) err
...
+	return allErrs
+}
+
+// validateIDRanges ensures the range is valid.
+func validateIDRanges(rng []api.IDRange) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+
+	for i, r := range rng {
+		if r.Min > r.Max {
+			field := fmt.Sprintf("ranges[%d]", i)
```
Comment (pmorie) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation.go]: .max missing?

Comment (pweil-) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation.go]: this is indicating the # element in the range array.  Not sure the question

Comment (pmorie) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation.go]: Nevermind, I get it after we chatted about it, I think a comment would help here.

Comment (pweil-) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/validation/validation.go]: will comment it up.

**Pull request merged**